### PR TITLE
Tony/umi

### PR DESCRIPTION
**Pull request by tonyboylehub**

### CODE REVIEW

File: src/pages/umi/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+A Solana Framework for JavaScript clients.
+
+Umi is a modular framework for building and using JavaScript clients for Solana programs. It provides a zero-dependency library that defines a set of core interfaces that libraries can rely on without being tied to a specific implementation. It is then up to the end-user to choose the implementation that best suits their needs. Umi also provides a set of default implementations and bundles that can be used out of the box allowing developers to get started quickly. {% .lead %}
+
+{% quick-links %}
+
+{% quick-link title="Getting Started" icon="InboxArrowDown" href="/token-metadata/getting-started" description="Find the language or library of your choice and get started essentials programs." /%}
+
+{% quick-link title="API reference" icon="CodeBracketSquare" href="/token-metadata/references" description="Looking for something specific? Have a peak at our API References and find your answer." /%}
```
Comment (MarkSackerberg) [inline src/pages/umi/index.md:15]:
```suggestion
{% quick-link title="Getting Started" icon="InboxArrowDown" href="/umi/getting-started" description="Find the language or library of your choice and get started essentials programs." /%}

{% quick-link title="API reference" icon="CodeBracketSquare" href="/umi/references" description="Looking for something specific? Have a peak at our API References and find your answer." /%}
```

File: src/components/products/umi/index.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            { title: 'Web3js Differences', href: '/umi/web3js-differences' },
+            
+          ],
+        },
+        {
+          title: 'Features',
+          links: [
+            { title: 'Accounts', href: '/umi/accounts' },
+            { title: 'Helpers', href: '/umi/helpers' },
+            { title: 'Http Requests', href: '/umi/http-requests' },
```
Comment (MarkSackerberg) [inline src/components/products/umi/index.js:39]:
```suggestion
            { title: 'HTTP Requests', href: '/umi/http-requests' },
```

**Review by MarkSackerberg — COMMENTED** [review]
image
Is there a reason that you are not using the dialect switcher? Sure, we only have umi / JS as dialect here, but the headings look nicer from my POV than using the Komment as header
image

File: src/pages/umi/changelog/index.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: Changelog
+metaTitle: Toolbox - Changelog
```
Comment (MarkSackerberg) [inline src/pages/umi/changelog/index.md:3]:
```suggestion
metaTitle: Umi - Changelog
```

**Comment by blockiosaurus** [comment]
> image
> Is there a reason that you are not using the dialect switcher? Sure, we only have umi / JS as dialect here, but the headings look nicer from my POV than using the Komment as header
> image
>  

Did this get addressed? At a glance everything looks good to me but not sure what specifically you were calling out.

**Comment by tonyboylehub** [comment]
> > image
> > Is there a reason that you are not using the dialect switcher? Sure, we only have umi / JS as dialect here, but the headings look nicer from my POV than using the Komment as header
> > image
> 
> Did this get addressed? At a glance everything looks good to me but not sure what specifically you were calling out.

Yeah basically swapped out the //comment headers for the code block headers like in image 1 that Mark shared.

**Review by blockiosaurus — APPROVED** [review]

**Review by blockiosaurus — APPROVED** [review]

**Review by blockiosaurus — APPROVED** [review]

**Pull request merged**